doc/src/sgml/func3.sgml

```diff
@@ -615,7 +615,7 @@ JSON値が配列でなければエラーが生じます。
         <link linkend="functions-sqljson-check-expressions">predicate check
         expressions</link>, since those always return a value.)
 -->
-《機械翻訳》JSONパスは指定されたJSON値に対して何らかの項目を返しますか?（これはSQL標準のJSONパス式でのみ有用であり、<link linkend="functions-sqljson-check-expressions">述語チェック式</link>では値を返しません。）
+JSONパスは指定されたJSON値に対して何らかの項目を返すか?（これはSQL標準のJSONパス式でのみ有用であり、<link linkend="functions-sqljson-check-expressions">述語チェック式</link>では値を返しません。）
```

Comment:
機械翻訳が誤っているのはそのとおりだと思います。ただ、since...は「...だから」と理由を述べていると思うので、ご提案の訳では違うと思います。以下のようにしました。
`（これはSQL標準のJSONパス式でのみ有用であり、<link linkend="functions-sqljson-check-expressions">述語チェック式</link>では有効ではありません。述語チェック式は常に値を返すからです。）`